.github/workflows/ci.yml

```diff
@@ -1524,7 +1524,7 @@ jobs:
           cargo run --bin uv -- pip compile scripts/requirements/airflow.in --universal --exclude-newer 2024-08-08 --cache-dir .cache
 
       - name: "Build benchmarks"
-        run: cargo codspeed build --profile profiling --features codspeed -p bench
```

Comment:
Was this intended to be removed?

Reply:
Yeah, but it's restored now that https://github.com/astral-sh/uv/pull/7276 is merged